src/components/Multiselect/Common/useMultiselect.tsx

```diff
@@ -59,9 +61,10 @@ export const useMultiselect = <T extends Option, V extends Option | string>({
 
   const itemsToSelect = getItemsFilter<T>(selectedItems, inputValue, options);
 
-  const showInput = onInputValueChange
-    ? true
-    : selectedItems.length !== options.length;
+  const showInput =
```

Comment:
**Suggestion:** It starts to look like nested ternary - can you refactor it to if statement?